PukingPredator/Assets/Scripts/PlayerState.cs

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
perhaps rather than making a separate class for this, we should start of with just converting the ConsumableState into a generic State class and then we can inherit from that if we need to add more specific behaviour for one of these cases down the road